.github/actions/setup_ubuntu_build/action.yml

```diff
@@ -24,10 +24,20 @@ runs:
   steps:
     - name: Install prerequisites
       shell: bash    
+      env:
+        UBUNTU_VER : '${{ inputs.ubuntu_version }}'
+        ARCH : '${{ inputs.arch }}'
       run: |
-        sudo apt-get install -y spirv-tools
-        pip install lit
-
+        pip install lit clang-format==17.0.6 virtualenv
+        if [ "$ARCH" = "x86" ]; then sudo dpkg --add-architecture i386 ; fi
+        wget -qO - https://packages.lunarg.com/lunarg-signing-key-pub.asc | sudo apt-key add -
+        if [ "$UBUNTU_VER" = "20.04" ]; then sudo wget -qO /etc/apt/sources.list.d/lunarg-vulkan-1.3.243-focal.list https://packages.lunarg.com/vulkan/1.3.243/lunarg-vulkan-1.3.243-focal.list; fi
```

Comment:
can't we just use the ${{ inputs.ubuntu_version }}, I can't see the point in adding an env which gives no more information.

Reply:
I think we can. I had syntax fails with an early version of this and found that there were known issues with the use of inputs in that context. Also some security hardening advice recommends using intermediate env vars rather than inputs (or any Githab vars) directly. However the issues look to be fixed and I have updated to use inputs.